packages/manager/modules/product-offers/src/pricing/extra-pricing-detail/extra-pricing-detail.html

```diff
@@ -0,0 +1,11 @@
+<span data-translate="ovh_product_offers_pricing_extra_pricing"></span>
+<strong
+    data-translate="{{('ovh_product_offers_pricing_capacity_' + $ctrl.pricingCapacity)}}"
+></strong>
+-
+<ovh-manager-order-catalog-price
```

Comment:
You are still linked to the `@ovh-ux/manager-web` application so it means that this module isn't self-contained.

```sh
$ git grep "ovhManagerOrderCatalogPrice"
packages/manager/apps/web/client/app/components/manager-order-catalog-price/index.js:const moduleName = 'ovhManagerOrderCatalogPrice';
packages/manager/apps/web/client/app/components/manager-order-catalog-price/index.js:  .component('ovhManagerOrderCatalogPrice', component)
```

Reply:
@antleblanc yes, there is the following PR for the resolution of this link : https://github.com/ovh/manager/pull/2336